stdlib/tests/crypto/falcon.rs

```diff
@@ -228,34 +163,41 @@ fn test_falcon512_probabilistic_product() {
     test.expect_stack(expected_stack);
 }
 
-#[ignore = "needs horner_eval_base op"]
 #[test]
 fn test_falcon512_probabilistic_product_failure() {
-    // Create a polynomial pi that is not equal to h * s2.
+    // create a polynomial pi that is not equal to h * s2.
```

Comment:
The `random_coefficients` use rng - can we either fix a seed, or generate those coefficients deterministically?